packs/travisci/actions/list_repos.py

```diff
@@ -0,0 +1,19 @@
+from lib.action import TravisCI
+import requests
+import yaml
+
+
+class GetRepos(TravisCI):
+    def run(self, username):
+        """
+        Listing all Repos for a give user
+        """
+        _HEADERS = self.travis
```

Comment:
You should be able to reduce the amount of duplicated code by refactoring this functionality into a new `_perform_request(url, query_params, method='GET')` or a similar method on the base action class.

This method should perform the request and return the parsed response.